infra.bs

```diff
@@ -119,6 +119,29 @@ will make the caller automatically rethrow the given value, if any, and thereby
 caller's algorithm. Using prose the caller has the ability to "catch" the exception and perform
 another action.
 
+<h4 id=algorithm-iteration>Iteration</h4>
+
+<p>There's a variety of ways to repeat a set of steps until a condition is reached. The Infra
+Standard is not (yet) exhaustive on this, but here are some examples:
+
+<ul>
+ <li><p>For each, as demonstrated later in this section.
+
+ <li>
+  <p>While
+
+  <div class=example id=example-while>
+   <ol>
+    <li>
+     <p>While true:
```

Comment:
This is not a great example because it's better rewritten as `While |condition| is "met"`. And unlike the foreach example, it doesn't actually do anything. And it uses break/continue before they are introduced. Maybe just an example like "while condition is met, perform operation" would be good for introducing while. Then we could do a separate "while true" + break example after introducing break/continue.